### DISCUSSION TITLE
How to send emails to contacts in database

### DISCUSSION THREAD

**Discussion by GrietVanbillemont** [Budibase/budibase#12801] [Help]
I have a database recieving information from a form. I want to send a personalized email to a group of contacts that filled in the form. (e.g. all answers with a specific value in a column should recieve an email). This email is not send after the row is created (this automation is already applied), it is an email that is send later.

**Comment by ConorWebb96** [comment]
Hey @GrietVanbillemont,

You can use an automation to send an email out to all the people who have answered questions with a specific value. You'll need to do some filtering, then with the filtered results you can send emails out to only the correct users.

- Heres the documentation for the send email step in [automations](https://docs.budibase.com/docs/send-email).
- Heres documentation on filtering [automation](https://docs.budibase.com/docs/data-in-automations#querying-data-with-trigger-data-as-filter) results.

Hopefully this helps.
